src/directive.js

```diff
@@ -19,7 +19,7 @@
       template: '<span ng-class="{\'is-placeholder\': placeholder && !editingValue}" ng-style="{\'max-width\': \'inherit\'}" >' +
         '<input ng-show="isEditing" ng-blur="onInputBlur()" ng-keydown="onKeyPress($event)" ' +
           'ng-model="editingValue" placeholder="{{placeholder}}" type="text" ' +
-          'pu-elastic-input pu-elastic-input-minwidth="auto" pu-elastic-input-maxwidth="100%" />' +
+          'pu-elastic-input pu-elastic-input-minwidth="100%" pu-elastic-input-maxwidth="100%" />' +
```

Comment:
I'm not sure what it could be adding either.  I can remove it and see what happens.